src/main/java/Exceptions/DukeCorruptedDataException.java

```diff
@@ -0,0 +1,8 @@
+package Exceptions;
```

Comment:
I like that you have a lot of exceptions, which makes it clearer to user what went wrong 😄